src/garage/torch/algos/sac.py

```diff
@@ -316,17 +308,17 @@ def _actor_objective(self, samples_data, new_actions, log_pi_new_actions):
         Note:
             samples_data's entries should be torch.Tensor's with the following
             shapes:
-                observation: :math:`(N, O^*)`
-                action: :math:`(N, A^*)`
-                reward: :math:`(N, 1)`
-                terminal: :math:`(N, 1)`
-                next_observation: :math:`(N, O^*)`
+                observations: :math:`(N, O^*)`
```

Comment:
a goal in this PR should be to remove samples_data and use TimeStepBatch

Reply:
In this case the input is a `torch.Tensor`, so `TimeStepBatch` can't be used. However, looking at the definition of this method, `SAC._get_log_alpha`, and the override `MTSAC._get_log_alpha`, only the observations are ever used. It would probably be best to refactor all three methods to only take the `observations` tensor.